### ISSUE TITLE
`with` used in java

### ISSUE THREAD

**Issue by hamoid** [driver733/kotlin-vs-java#108]
https://github.com/driver733/kotlin-vs-java/blob/7bfda57b1db1ff8b3b59a73f9169ac200f3abbd5/code/java/scoping-functions/with.java#L2

I'm I wrong in thinking that this does not work in Java?

**Comment by driver733** [comment]
@hamoid Yes, you are right. You are welcome to submit a PR to fix this error.
